### ISSUE TITLE
The Unsupervised denoising training example in T5's doc

### ISSUE THREAD

**Issue by drxmy** [huggingface/transformers#12905]
When I am running that example, it will print a lot of "seq_length: 7" like this:
seq_length: 7
position_bias: torch.Size([1, 8, 7, 7])
mask: torch.Size([1, 1, 1, 7])
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7
position_bias: torch.Size([1, 8, 7, 7])
mask: torch.Size([1, 1, 7, 7])
seq_length: 7
position_bias: torch.Size([1, 8, 7, 7])
mask: torch.Size([1, 1, 1, 7])
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7
seq_length: 7

But I do get the loss. If I run my own training code with a 256 sequence length and T5-large, it will print a lot more. Is this normal? My environments are:

1. torch 1.7.1
2. transformers 4.8.2
3. cuda 10.1
4. GPU v100-16g

Could you please help me with this issue? Thank you!

**Comment by NielsRogge** [comment]
I just ran it in a Colab notebook and got no issues.

Here's the notebook: https://colab.research.google.com/drive/1Fq420RZwq2coLjb0TJmAx5Q20uz3JJHj?usp=sharing

**Comment by drxmy** [comment]
> I just ran it in a Colab notebook and got no issues.
> 
> Here's the notebook: https://colab.research.google.com/drive/1Fq420RZwq2coLjb0TJmAx5Q20uz3JJHj?usp=sharing

Thank you for the trouble! I changed my torch to 1.9 and transformers to 4.9. It didn't happen anymore.
